servicetalk-log4j2-mdc/build.gradle

```diff
@@ -31,14 +31,20 @@ afterEvaluate {
 }
 
 dependencies {
+  api project(":servicetalk-concurrent-api")
   api project(":servicetalk-log4j2-mdc-utils")
 
   implementation project(":servicetalk-annotations")
+  implementation project(':servicetalk-context-api')
   implementation "org.apache.logging.log4j:log4j-core"
 
+
   testImplementation enforcedPlatform("org.apache.logging.log4j:log4j-bom:$log4jVersion")
   testImplementation enforcedPlatform("org.junit:junit-bom:$junit5Version")
   testImplementation "org.apache.logging.log4j:log4j-api"
   testImplementation "org.junit.jupiter:junit-jupiter-api"
   testImplementation "org.hamcrest:hamcrest:$hamcrestVersion"
+  testImplementation "org.slf4j:slf4j-api:$slf4jVersion"
```

Comment:
IIUC you can include the test-resources module instead then you get the log4j config for free too

Reply:
I still need to import this or else I get compile errors, but I did include test resources and remove the config.